eng/testing/gen-debug-dump-docs.py

```diff
@@ -86,15 +86,19 @@
 
 replace_string = ''
 dir_separator = '/' if platform.system() != 'Windows' else '\\'
+unix_user_folder = '~/dumps/'
+windows_user_folder = 'c:\\dumps\\'
```

Comment:
instead of using two new variables and a placeholder for windows and linux, why not just use `platform.system` to set the path to `~/.../` or `c:\\dumps\\`.

Reply:
That's a good idea. I'm going to let this merge in as I think it's OK and it will be a while before I have a chance to retest it if I change it. I will make a note to improve this in future.

Reply:
(If you're OK with that of course)

Reply:
Yeah, I'm OK with it, it was a small nit that jumped out, but anyway is fine 😄